docs/docs/end-to-end-testing.md

```diff
@@ -1,16 +1,16 @@
 ---
-title: "End-to-end testing"
+title: End-to-end testing
```

Comment:
```suggestion
title: End-to-End Testing
```

While we're updating this, let's make it adhere to the Style Guide.

Reply:
Sure. Do you have more suggestions or should I commit this?

Reply:
I do, but I had to jump into a meeting. Stay tuned